Makefile

```diff
@@ -7,36 +7,54 @@
 # Variables
 #--------------------------------------------------
 PYTHON_FILES?=$$(find biomage -name '*.py')
-INSTALL_PATH=/usr/local/bin
-MKFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
-MKFILE_DIR := $(dir $(MKFILE_PATH))
+ENTRY_POINT=/usr/bin/biomage
```

Comment:
you cannot install into `/usr/bin` in mac because it's protected by SIP. Either detect if the OS is linux / mac and install into different directories or just add a note in the `Readme.md` telling linux users to add it to the PATH.

Reply:
Will do the former.

Reply:
there's an example in some other makefile iirc so it's consistent, if you find a better way we can change the other

Reply:
This should be fixed now!